lib/glueby/internal/wallet.rb

```diff
@@ -91,6 +91,21 @@ def create_pubkey
         wallet_adapter.create_pubkey(id)
       end
 
+      def collect_uncolored_outputs(amount)
+        utxos = list_unspent
+
+        utxos.inject([0, []]) do |sum, output|
+          next sum if output[:color_id]
+
+          new_sum = sum[0] + output[:amount]
+          new_outputs = sum[1] << output
+          return [new_sum, new_outputs] if new_sum >= amount
+
+          [new_sum, new_outputs]
+        end
+        raise Glueby::Contract::Errors::InsufficientFunds
+      end
+
```

Comment:
This code almost duplicate with `TxBuilder#collect_uncolored_outputs`.

Reply:
Which do you think is better, to remove `TxBuilder#collect_uncolored_outputs` or to modify it as below?
```
      def collect_uncolored_outputs(amount)
        utxos = list_unspent

        Glueby::Contract::TxBuilder.new.collect_uncolored_outputs(utxos, amount)
      end
```

Please let me know if there is another good way.🙏

Reply:
It is better to remove `TxBuilder#collect_uncolored_outputs` and use the wallet one.